prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,13 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+languages.firstElement = 'html';
+console.log('the first element of the array is', languages.firstElement);
+
+languages.thirdElement = 'javascript';
+console.log('the 3rd element of the array is', languages.thirdElement);
```

Comment:
We can access the values of elements in an array using their indexes.

```js
var firstElement = languages[0];
```

Reply:
![prep-javascript-arrays](https://user-images.githubusercontent.com/113574704/192661617-b2f3540a-99e0-43b3-98f8-19b70d7121e1.PNG)